src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -51,6 +53,15 @@
 @ToString
 @EqualsAndHashCode(of = "profile")
 final class Decrypt {
+    /**
+     * Name of the system property, which specifies HTTP proxy host.
+     */
+    public static final String HTTP_PROXY_HOST = "http.proxyHost";
```

Comment:
I think it's a wrong design. This approach makes this class NOT thread-safe. Try to avoid static methods/variables at all cost. A better design would be to pass host/port as constructor arguments to the `Decrypt` class.